website_product_brand/__manifest__.py

```diff
@@ -10,7 +10,7 @@
     'license': 'AGPL-3',
     'summary': 'Product Brand based filters',
     'version': '10.0.0.0.1',
-    'depends': ['product_brand', 'website_sale'],
+    'depends': ['product_brand', 'website_sale','sale_management'],
```

Comment:
website_sale will always do what you did. No need to specify explicitly.
If you are making a PR in v10, check it wasnt sale_management .

Reply:
I think it is the product_brand plugin that requires it.   To import product brands the menu is in the sales app.